README.md

```diff
@@ -0,0 +1,28 @@
+# NLQ Notebooks
+
+This repository provides python modules and Jupyter Notebooks demonstrating a Natural Language Querying (NLQ) system using [GraphDB](https://graphdb.ontotext.com/) and 
+[OpenAI Assistants API](https://platform.openai.com/docs/api-reference/assistants) or
+[LangGraph Agents](https://langchain-ai.github.io/langgraph/how-tos/create-react-agent/).
+The LangGraph implementation can be used to plug in any open-source Large Language Model (LLM) including locally deployable ones.
+
+## Usage
+
+### Set up an environment with conda and poetry
+
+#### Prerequisites
+
+You should install [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html). `miniconda` will suffice.
+
+#### Create the environment
+
+```bash
+conda create --name nlq-notebooks --file conda-linux-64.lock
+conda activate nlq-notebooks
+poetry install --no-root
+```
+
+### Run
+
```

Comment:
Should we make it explicit that a GraphDB instance must be already running, filled with data and accessible? Or that alternatively the docker-compose can be used to start it?

Reply:
I've added this section in the Jupyter notebooks
![image](https://github.com/user-attachments/assets/988fb6a6-2e9b-40eb-9460-158a5306b1a2)